core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -1046,7 +1047,7 @@ public PropServerConfiguration(
             getUrls(properties, env, PropertyKey.HTTP_CONTEXT_EXPORT, this.httpContextPathExport, httpContextWebConsole + "/exp");
             getUrls(properties, env, PropertyKey.HTTP_CONTEXT_SETTINGS, this.httpContextPathSettings, httpContextWebConsole + "/settings");
             getUrls(properties, env, PropertyKey.HTTP_CONTEXT_TABLE_STATUS, this.httpContextPathTableStatus, httpContextWebConsole + "/chk");
-            getUrls(properties, env, PropertyKey.HTTP_CONTEXT_EXECUTE, this.httpContextPathExec, httpContextWebConsole + "/exec");
+            getUrls(properties, env, PropertyKey.HTTP_CONTEXT_EXECUTE, this.httpContextPathSqlExecute, httpContextWebConsole + "/exec", httpContextWebConsole + "/api/v1/sql/execute");
```

Comment:
nit: shouldn't we add `/api/v1/sql` for `/exp`, `/imp` and other existing endpoints? What's the purpose of having the new path only for `/exec` and `/validate`?

Reply:
> What makes this the priority?

Sorry, I'm confused. Are you asking what makes a nit comment the priority or it's a question for Nick?